src/domain/cohort.js

```diff
@@ -21,16 +21,36 @@ export async function getCohorts() {
   return Cohorts
 }
 
-export class Cohort {
+export default class Cohort {
```

Comment:
It may not be a good idea to export this as a default as there are functions also being exporting in this file. It won't cause any problems other than the import being slightly harder to read. I'll leave this down to your preference as to if you want to change it.